assembl/static2/js/app/utils/csrf.js

```diff
@@ -0,0 +1,11 @@
+import request from 'superagent';
+import urljoin from 'url-join';
+
+import { basePath } from './server';
+
+export async function getCSRFToken() {
+  // Get CSRF token, note the response contains nothing as the cookie is set directly!!
+  const responseToken = await request.get(urljoin(basePath(), '/v1/generateToken')).withCredentials();
```

Comment:
Is the logic for `/v1/generateToken` hosted on gitlab? Possible open a PR for that, or at least link to the project so that we can CR that too. I think it's important to look at that code too.

Reply:
It's stargate and committed to master

Reply:
Super, I'll take a look at stargate code as well. Anyway to open up a PR there? Or at least, I can look at it and give you my impressions. It has no bearing on this PR though.

Reply:
Please do take a look at Stargate - no review because I looked around and asked myself who here is better than me at javasccript :-)